todo_app/app.py

```diff
@@ -1,11 +1,27 @@
-from flask import Flask
+from flask import Flask, render_template
+from flask import request
+from todo_app.data.session_items import get_items
+from todo_app.data.session_items import add_item
+
+from todo_app.flask_config import Config 
 
-from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
-@app.route('/')
+@app.route('/', methods=['GET', 'POST'])
 def index():
-    return 'Hello World!'
+    items = get_items()
+    return render_template('index.html', items = items)
+
+@app.route('/todo', methods=['GET', 'POST']) 
+def addItem(): 
+        items = get_items()
+        
+        new_item = request.form.get('Todo')
+        add_item(new_item)
+        
+       
+        return render_template('index.html', items = items)
```

Comment:
This works, but it leads you with a couple of side-effects:
* As a maintainer of this code, any time the `index` function changes, we're going to also have to update this function
* It relies on the fact that the `items` variable updates when `add_item` is called (which isn't obviously always going to be true!)
* It means that users who refresh the page accidentally try to submit the same item again, or similarly for bookmarking the page after hitting this endpoint

All of those can be solved by switching this to instead redirect to the index function - that will keep those behaviours disconnected and make our job a little easier 🙂 
That will involve importing the `redirect` function from Flask and then using it like:
```
return redirect('/')
```